Documentation/ProjectFile/prj/processes/process/THERMO_MECHANICS/constitutive_relation/i_constitutive_relation.md

```diff
@@ -0,0 +1 @@
+The constitutive relation for the mechanical part. Only linear elasticity is available in the moment.
```

Comment:
why is that? If we pass mechanical strain into compute constitutive, it should work for all material models.